lib/index.ts

```diff
@@ -4,7 +4,7 @@ require('./nconf-truth');
 var path = require('path');
```

Comment:
for the ```BREAKING CHANGE``` commits - I think the commit message needs to change
![image](https://user-images.githubusercontent.com/1255387/69061388-ab28d580-0a21-11ea-8407-d9b1d9683bce.png)
(from https://www.notion.so/snyk/Release-Publish-and-Deploy-Process-7a731eb98f0745d0b808ace62f666afb)

Reply:
so ```BREAKING CHANGE: export a named function```
"should be"

```
fix: export a named function
BREAKING CHANGE: :goose:
```

(not sure if we need a line break between the first and the second)

Reply:
I think you're correct.

I'd convinced myself it was required to do this way.

Fixed.

Reply:
Well balls, I managed to not push that fix.